types/applications/sidebarTabs/sidebarDirectories/playlistDirectory.d.ts

```diff
@@ -0,0 +1,84 @@
+/**
+ * A directory listing of audio playlists
+ */
+declare class PlaylistDirectory extends SidebarDirectory {
+  constructor(options?: Partial<Application.Options>);
+
+  /**
+   * Track the playlist IDs which are currently expanded in their display
+   */
+  protected _expanded: Set<string>;
+
+  /**
+   * @override
+   */
+  static get defaultOptions(): PlaylistDirectory.Options;
+
+  /**
+   * @override
+   */
+  static get entity(): string;
+
+  /**
+   * @override
+   */
+  static get collection(): Playlists;
+
+  /**
+   * Return an Array of the Playlist entities which are currently playing
+   */
+  get playing(): Playlist[];
+
+  /**
+   * Prepare the data used to render the AudioList application
+   * @param options - (unused)
+   */
+  getData(options?: Application.RenderOptions): PlaylistDirectory.Data;
+
+  /**
+   * Given a constant playback mode, provide the FontAwesome icon used to display it
+   */
+  protected _getModeIcon(mode: Const.PlaylistModes): string;
+
+  /**
+   * Given a constant playback mode, provide the string tooltip used to describe it
+   */
+  protected _getModeTooltip(mode: Const.PlaylistModes): string;
+}
```

Comment:
Missing a couple of methods:
- `activateListeners`
- `_onGlobalVolume`
- `_onPlaylistCollapse`
- `_collapse`
- `_onPlaylistEdit`
- `_onPlaylistDelete`
- `_onPlaylistAddTrack`
- `_onPlaylistPlay`
- `_onPlaylistToggleMode`
- `_onSoundEdit`
- `_onSoundPlay`
- `_onSoundVolume`
- `_onSoundToggleMode`
- `_onSoundDelete`
- `_contextMenu`
- `_getSoundContextOptions`